### PR TITLE
Update for webpack5

### PR DESCRIPTION
**Pull request by christianopaets**
This can fix errors with the newest version of html-webpack-plugin and webpack5

### CODE REVIEW

**Review by Epimodev — CHANGES_REQUESTED** [review]
@christianopaets thanks a lot for this pull request!
As it's been a long time I didn't use webpack, I didn't try yet if it works well. But I've already a few suggestions. If you disagree don't hesitate to tell me and explain why :-)

**Comment by christianopaets** [comment]
Thank you for your answer. I've made some changes and tested with old version of webpack and html-webpack-plugin. Now it works fine)))

File: src/plugin.js [hunk 1]
```diff
@@ -176,17 +206,17 @@ module.exports = class SvgSpriteHtmlWebpackPlugin {
    */
   applyWebpack4(compiler) {
```
Comment (Epimodev) [inline src/plugin.js:207]: great you've made something better than I imagine 👍 
I just think this function should be named just `applyWebpack` as it works for webpack 4 and 5.

Thanks to have tested it with old version 🙏
I let you make this little change and I will merge and release the new version this afternoon

Comment (christianopaets) [reply src/plugin.js]: done

**Review by Epimodev — APPROVED** [review]
Perfect thanks a lot!

**Comment by Epimodev** [comment]
@christianopaets I just published version 2.4.0 with your changes

**Comment by christianopaets** [comment]
@Epimodev thanks a lot

**Pull request merged**